src/logger.cc

```diff
@@ -56,12 +46,23 @@ NAN_METHOD(setLevel) {
   spdlog::set_level(level);
 }
 
+NAN_METHOD(setFlushEvery) {
```

Comment:
We're currently using an async logger with a flush interval of 500ms in VS Code. According to https://spdlog.docsforge.com/v1.x/7.flush-policy/, if we still want to use interval-based flushing, I can switch back to using `_mt` loggers, since the flushing requires thread-safe loggers to work.

Should we continue using interval-based flushing?

Reply:
`set_async_mode` was needed when a logger can be switched between sync and async, it allowed to specify the flush interval and block policy. I would suggest to rely on the default flush interval set by `spdlog` for `_st` loggers and if needed we can always do a manual flush.